Core/clim-basic/views.lisp

```diff
@@ -44,34 +44,6 @@
 (defclass pointer-documentation-view (textual-view)
   ())
 
-;;; Views described in the Franz User manual (CLIM 2.2)...
-
-(defclass toggle-button-view (gadget-view)
-  ())
-
-(defclass push-button-view (gadget-view)
-  ())
-
-(defclass radio-box-view (gadget-view)
-  ())
-
-(defclass check-box-view (gadget-view)
-  ())
-
-(defclass slider-view (gadget-view)
-  ())
-
-(defclass text-field-view (gadget-dialog-view)
-  ((width :accessor width :initarg :width :initform nil)))
```

Comment:
Please explain in the commit message:

* Why can the `width` slot be removed? (The `define-gadget-view` macro looks at `text-field` which doesn't have such a slot)
* Why is the superclass now `gadget-dialog-view` for all view classes when it previously was `gadget-view` for most view classes?